crates/compiler/src/compiler/compiler.rs

```diff
@@ -197,15 +199,29 @@ impl Compiler for DojoCompiler {
 
         // Combine the aux data info about the contracts with the artifact data
         // to create the manifests.
-        let contracts = write_dojo_contracts_artifacts(&artifact_manager, &aux_data)?;
-        let models = write_dojo_models_artifacts(&artifact_manager, &aux_data)?;
+        let contracts = write_dojo_element_artifacts::<ContractAuxData, ContractManifest>(
```

Comment:
```suggestion
        let contracts = write_dojo_resource_artifacts::<ContractAuxData, ContractManifest>(
```
Generally speaking, contracts, models and events are resources. Would make sense to rename as such. :)

Reply:
Ah yes 👍